en/scripting/typescript.md

```diff
@@ -162,7 +172,24 @@ export class MyUser extends cc.Component {
 
 When you enter `this.myModule.`, you will be prompt the properties declared in `MyModule.ts`.
 
-![Auto complete](assets/auto-complete.gif)
+![Auto complete](assets/auto-complete.gif) <br>
+__If you find that changing a declared property to an array type doesn't take effect in the editor, Please remove the binding component from the current node and rebind.__
+
+## The special type since 1.10 version
+
+The Creator has adjusted inheritance relationship of several basic class after the 1.10 version.
+You need to modify the statement that declares the cc.Texture2D, cc.AudioClip and cc.ParticleAsset as below description
```

Comment:
Cocos Creator made partial adjustments to the asset types since v1.10. The declaration of data type `cc.Texture2D`, `cc.AudioClip` and `cc.ParticleAsset` etc. in TS must be declared in the following format: